ouroboros-network-framework/src/Ouroboros/Network/InboundGovernor.hs

```diff
@@ -501,6 +504,27 @@ runResponder mux
             startStrategy
             (runMuxPeer responder . fromChannel)
 
+--
+-- Auxiliary Types
+--
+
+-- | A Server control channel which instantiates to 'NewConnection' and 'Handle'.
+--
+-- It allows to pass 'STM' transactions which will resolve to 'NewConnection'.
+-- Server's monitoring thread is the consumer of these messages; there are two
+-- producers: accept loop and connection handler for outbound connections.
+--
+type InboundGovernorInfoChannel (muxMode :: MuxMode) peerAddr versionData bytes m a b =
+    InformationChannel (NewConnectionInfo peerAddr (Handle muxMode peerAddr versionData bytes m a b)) m
+
+-- | Control Channel between Server and Outbound Governor.
+--
+-- Control channel that is meant to share inbound connections with the Peer
+-- Selection Governor. So the consumer is the Governor and Producer is the
+-- Server.
+--
+type OutboundGovernorInfoChannel peerAddr m =
+    InformationChannel (peerAddr, PeerSharing) m
```

Comment:
I think a better place for these two type aliases is the module where `InformationChannel` is defined.  It's a bit confusing why `OutboundGovernorInfoChannel` is defined in `Ouroboros.Network.InboundGovernor`.